ALUM/ALUM/Components/ParagraphInput.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ParagraphInput.swift
+//  ALUM
+//
+//  Created by Jenny Mar on 2/15/23.
+//
+
+// call back function exists in parent class
+// when its called, it saves the text into the small paragraph
+// the method is called when done is pressed
+
+import SwiftUI
+
+struct ParagraphInput: View {
+    @State var question: String
+    @Binding var text: String
+    @State private var totalChars = 0
+    @State private var lastText = ""
+
+    var body: some View {
+        HStack {
+            Text(question)
+                .font(.system(size: 17))
+                .foregroundColor(Color("ALUM Dark Blue"))
+            Spacer()
+        }
+        TextEditor(text: $text)
+            .padding(20)
+            .frame(height: 410)
+            .cornerRadius(12)
+            .overlay(
+                RoundedRectangle(cornerRadius: 12)
+                    .stroke(Color("NeutralGray3"), lineWidth: 1))
+            .colorMultiply(Color("ALUM White"))
+            .background(Color("ALUM White"))
+            .onChange(of: text, perform: { text in
+                totalChars = text.count
+                if totalChars <= 1000 {
+                        lastText = text
+                    } else {
+                        self.text = lastText
+                    }
```

Comment:
```suggestion
                if text.count > 1000 {
                    self.text = String(text.prefix(1000))
                }
```
This way we don't introduce any new states. It's always nicer and easier to work with lesser states when your code base becomes huge